src/Components/Browser.JS/src/Platform/Circuits/AutoReconnectCircuitHandler.ts

```diff
@@ -17,15 +20,15 @@ export class AutoReconnectCircuitHandler implements CircuitHandler {
       }
     });
   }
-  onConnectionUp()  : void{
+  public onConnectionUp(): void {
```

Comment:
Why make this and the other methods public?

Reply:
https://github.com/aspnet/AspNetCore/pull/8911#discussion_r270841893
> These are changes are about making eslint/jest happy. ESlint complained about the lack of modifier and when I tried private, jest complained about it being private, so no other option but to make them public.
> 
> Right now there's no way for a customer to access AutoReconnectCircuitHandler, so I don't think it's a big deal.